src/core/public/core_app/core_app.ts

```diff
@@ -56,7 +60,20 @@ export class CoreApp {
         // Do not use an async import here in order to ensure that network failures
         // cannot prevent the error UI from displaying. This UI is tiny so an async
         // import here is probably not useful anyways.
-        return renderApp(params, { basePath: http.basePath });
+        return renderErrorApp(params, { basePath: http.basePath });
+      },
+    });
+
+    if (injectedMetadata.getAnonymousStatusPage()) {
+      http.anonymousPaths.register('/status');
```

Comment:
You cannot navigate to the other Kibana pages from other `anonymous pages` without reloading. It creates a bug when `status.allowAnonymous: true` and you logged-in user lands on the `/status` page and navigates to other pages. result: `SecurityNavControl` is not rendered.
![2020-07-22_10-50-55](https://user-images.githubusercontent.com/3198181/88157348-f1ff4880-cc0a-11ea-970f-481a79fb241b.png)
As a workaround, we could make `status` page chromeless to enforce page reloading.
Is it possible to land on the `/status` page from a link in Kibana UI?

Reply:
> You cannot navigate to the other Kibana pages from other anonymous pages without reloading. It creates a bug when status.allowAnonymous: true and you logged-in user lands on the /status page and navigates to other pages. result: SecurityNavControl is not rendered.

Hum. This indeed is an issue. We really need to have a way to know about anonymous vs logged-in apps from core, to be able to automatically performs that kind of reload when using `navigateToApp` between anon/logged or logged/anon apps. Do you know if we have an issue for that?

> As a workaround, we could make status page chromeless to enforce page reloading.

chromeless apps do not really force page reloading, but I guess hiding the chrome navigation will remove any way for the user to navigate away from the status page using the UI. do you think it would be sufficient?

> Is it possible to land on the /status page from a link in Kibana UI?

Not that I'm aware of. The app is hidden and there is no link to it anywhere in chrome.

Reply:
> but I guess hiding the chrome navigation will remove any way for the user to navigate away from the status page using the UI. 

that was my reasoning as well. I think it's okay since the user navigates to the `/status` page manually.

> Hum. This indeed is an issue. We really need to have a way to know about anonymous vs logged-in apps from core, to be able to automatically performs that kind of reload when using navigateToApp between anon/logged or logged/anon apps. Do you know if we have an issue for that?

I'm not aware of any.